src/main/java/com/gamegoo/service/manner/MannerService.java

```diff
@@ -252,9 +292,78 @@ public MannerRating update(MannerRequest.mannerUpdateDTO request, Long memberId,
                     mannerRatingKeyword.setMannerKeyword(mannerKeyword);
                 }
             }
+
+            // 매너점수 산정.
+            int mannerScore = updateMannerScore(targetMember);
+
+            // 매너레벨 결정.
+            int mannerLevel = mannerLevel(mannerScore);
+
+            // 매너레벨 반영.
+            targetMember.setMannerLevel(mannerLevel);
+
+            // db 저장.
+            memberRepository.save(targetMember);
+
             return mannerRatingRepository.save(mannerRating);
         }
+    }
+
+    // 매너점수를 산정하고 업데이트.
+    public int updateMannerScore(Member targetMember){
+
+        // 매너평가 ID 조회
+        List<MannerRating> mannerRatings = mannerRatingRepository.findByToMemberId(targetMember.getId());
+
+        int totalCount;
+
+        // 매너 평가 + 비매너 평가를 처음 받은 회원
+        if (mannerRatings.size()==1){
+            if (mannerRatings.get(0).getIsPositive()) {
+                totalCount = mannerRatings.get(0).getMannerRatingKeywordList().stream()
+                        .map(mannerRatingKeyword -> mannerRatingKeyword.getMannerKeyword().getId())
+                        .collect(Collectors.toList()).size();
```

Comment:
여기에서 어차피 mannerRatingKeyword 개수만 필요한 거라면, 굳이 map으로 id 변환시키지 않고 바로 list의 size만 뽑아도 될 것 같아요!
```suggestion
                totalCount = mannerRatings.get(0).getMannerRatingKeywordList().size();
```